drivers/sensor/sensirion/scd4x/scd4x.c

```diff
@@ -119,7 +185,6 @@ static int scd4x_attr_get(const struct device *dev,
     }
 
     if (attr == (enum sensor_attribute)SCD4X_ATTR_SERIAL_NUMBER) {
-        /* Read device ID */
         uint16_t device_id[3];
```

Comment:
Comment format will be addressed in an other issue.

```suggestion
        /* Read device ID */
        uint16_t device_id[3];
```